crates/rpc/rpc-eth-api/src/helpers/call.rs

```diff
@@ -259,31 +259,24 @@ pub trait EthCall: Call + LoadPendingBlock {
         let precompiles = get_precompiles(env.handler_cfg.spec_id);
         let mut inspector = AccessListInspector::new(initial, from, to, precompiles);
 
-        let (result, env) = self.inspect(&mut db, env, &mut inspector)?;
+        let (_, mut env) = self.inspect(&mut db, env, &mut inspector)?;
```

Comment:
this now ignores the result of the initial call, I'd like to keep that and only apply the transact after the match.

I'm fine with having another match result for the transact call